File: src/main/java/net/frozenblock/configurableeverything/biome/util/ParameterListExtension.java
package net.frozenblock.configurableeverything.biome.util;

import net.minecraft.core.RegistryAccess;
import net.minecraft.resources.ResourceKey;
import net.minecraft.world.level.dimension.DimensionType;

public interface ParameterListExtension {

	void updateBiomesList(RegistryAccess registryAccess, ResourceKey<DimensionType> dimension);
}

File: src/main/java/net/frozenblock/configurableeverything/biome_placement/util/ParameterListExtension.kt
package net.frozenblock.configurableeverything.biome_placement.util;

import net.minecraft.core.RegistryAccess;
import net.minecraft.resources.ResourceKey;
import net.minecraft.world.level.dimension.DimensionType;

public interface ParameterListExtension {

	void updateBiomesList(RegistryAccess registryAccess, ResourceKey<DimensionType> dimension);
}
